src/components/core/geoChart/geoChartTooltip.ts

```diff
@@ -132,7 +138,21 @@ export const handlePushpinMouseEnter = (
         .addTo(chart);
 };
 
-export const handlePushpinMouseLeave = (chart: mapboxgl.Map, tooltip: mapboxgl.Popup) => (): void => {
+export const handlePushpinMouseLeave = (
+    _e: mapboxgl.EventData,
+    chart: mapboxgl.Map,
+    tooltip: mapboxgl.Popup,
+    config: IGeoConfig,
+) => {
+    if (isTooltipDisabled(config)) {
+        return;
+    }
     chart.getCanvas().style.cursor = "";
     tooltip.remove();
 };
+
+// Tooltips are now switched off in edit/export mode
+function isTooltipDisabled(config: IGeoConfig): boolean {
+    const { viewport } = config;
+    return get(viewport, "frozen", false);
+}
```

Comment:
It could be
```
function isTooltipDisabled(config: IGeoConfig): boolean {
    const { viewport = {} } = config;
    return Boolean(viewport.frozen);
}
```

Reply:
updated